tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "noImplicitAny": false,
+    "allowJs": true,
+    "sourceMap": true,
+    "target": "es5"
```

Comment:
I'll probably change this to something newer, because babel takes care of downgrading the js at the end.

Reply:
Some of our testing suggests es2018 is reasonable